src/Loader/loader.scss

```diff
@@ -0,0 +1,106 @@
+@import "~@sebgroup/bootstrap/scss/core";
+@import "~@sebgroup/bootstrap/scss/styles/spinners";
+
+@mixin setLoaderSize($size) {
+    > svg {
+        width: $size;
+        height: $size;
+    }
+}
+
+$offset: 200;
+$duration: 1.5s;
+$duration-sq: 1.2s;
+.rc.loader {
+    color: $green;
+    &:not(.loader-cover):not(.loader-fullscreen) {
+        display: inline-flex;
+        vertical-align: middle;
+    }
+    &.loader-cover,
+    &.loader-fullscreen {
+        left: 0;
+        right: 0;
+        bottom: 0;
+        top: 0;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        flex-direction: column;
+        z-index: $zindex-tooltip + 10;
+    }
+    &.loader-cover {
+        position: absolute;
+    }
+    &.loader-fullscreen {
+        position: fixed;
+    }
+    &.loader-backdrop {
+        background-color: rgba($black, 0.55);
+    }
+    &.loader-xs {
+        @include setLoaderSize(16px);
+    }
+    &.loader-sm {
+        @include setLoaderSize(34px);
+    }
+    &.loader-md {
+        @include setLoaderSize(65px);
+    }
+    &.loader-lg {
+        @include setLoaderSize(100px);
+    }
```

Comment:
Can we agree to use `rem` where it makes sense? 
The bootstrap one is also using `rem` makes sense to just follow the same.

Reply:
You're right